app/assets/stylesheets/layouts/_call_to_action.sass

```diff
@@ -0,0 +1,5 @@
+.sign-up-cta
```

Comment:
Since the name of the partial is `call_to_action`, I believe the name of the class should be `call-to-action` too ;)